maths/area.py

```diff
@@ -481,3 +524,6 @@ def area_rhombus(diagonal_1: float, diagonal_2: float) -> float:
     print(f"Cone: {surface_area_cone(10, 20) = }")
     print(f"Conical Frustum: {surface_area_conical_frustum(10, 20, 30) = }")
     print(f"Cylinder: {surface_area_cylinder(10, 20) = }")
+    print(f"Equilateral Triangle: {area_reg_polygon(3, 10) = }")
+    print(f"Square: {area_reg_polygon(4, 10) = }")
+    print(f"Reqular Pentagon: {area_reg_polygon(5, 10) = }")

```

Comment:
You don't need them since you already used doctest to test your code.

Reply:
Hi @ZeroDayOwl , Based on the format of preexisting functions, I doubt if I'm allowed to proceed with your suggestion.